src/verify.test.ts

```diff
@@ -61,3 +63,56 @@ describe("verify", () => {
     });
   });
 });
+
+describe("verifyWithIndividualChecks", () => {
```

Comment:
what does this test bring compare to integration test ?

Reply:
The integration test ensures the checks work on an actual document with the correct network specified (also ensures the various checks work properly together).

The unit test, on the other hand, mocks what's returned by the various checks and checks whether the returned values are correct given these mocks. Also, we can mimic the way the various promises resolve at different times.

How should we better spilt the test concerns?

Reply:
As discussed, I've removed the unit tests that don't value add much.